src/utils/isValidInputEventName.js

```diff
@@ -0,0 +1,17 @@
+const VALID_EVENTS = [
+  'change',
+  'input',
+  'keyup',
+  'keydown',
+  'focus',
+  'focusin',
+  'focusout',
+  'click',
+];
+
+export default function isValidInputEventName(eventName) {
+  if(!eventName) return false;
+
+  let eventNameSanitized = eventName.toLowerCase().trim();
```

Comment:
let => const

toLowerCase można wywoływać tylko na stringu, więc albo rzutować to do stringa przed wywołaniem, albo wyżej w tym guard ifie sprawdzać czy to na pewno string

```suggestion
  const eventNameSanitized = eventName.toString().toLowerCase().trim();
```